docs/guides/volumesnapshot/pvc/index.md

```diff
@@ -135,263 +140,406 @@ spec:
-backupconfiguration.stash.appscode.com/pvc-volume-snapshot patched
-```
-
-Now, wait for a moment. Stash will pause the BackupConfiguration. Verify that the BackupConfiguration  has been paused,
-
-```bash
-$ kubectl get backupconfiguration -n demo
-NAME                  TASK   SCHEDULE      PAUSED   AGE
-pvc-volume-snapshot          */1 * * * *   true     22m
-```
-
-Notice the `PAUSED` column. Value `true` for this field means that the BackupConfiguration has been paused.
+This section will show you how to restore the PVC from the `VolumeSnapshot` we have taken in the earlier section.
 
 **Create RestoreSession :**
 
-At first, we have to create a `RestoreSession` crd to restore the PVC from respective snapshot.
-
-Below is the YAML of the RestoreSesion crd that we are going to create,
+Now, we are going to create a `RestoreSession` object to restore the `PVC` from respective `VolumeSnapshot`. Below is the YAML of the `RestoreSession` object that we are going to create,
 
 ```yaml
-apiVersion: stash.appscode.com/v1beta1
+apiVersion: core.kubestash.com/v1alpha1
 kind: RestoreSession
 metadata:
   name: restore-pvc
   namespace: demo
 spec:
-  driver: VolumeSnapshotter
-  target:
-    volumeClaimTemplates:
-    - metadata:
-        name: restore-data
-      spec:
-        accessModes: [ "ReadWriteOnce" ]
-        storageClassName: "csi-standard"
-        resources:
-          requests:
-            storage: 1Gi
-        dataSource:
-          kind: VolumeSnapshot
-          name: source-data-fnbwz
-          apiGroup: snapshot.storage.k8s.io
+  dataSource:
+    repository: gcs-repository
+    snapshot: latest
+  addon:
+    name: pvc-addon
+    tasks:
+      - name: VolumeSnapshotRestore
+        targetVolumes:
+          volumeClaimTemplates:
+            - metadata:
+                name: restore-source-pvc
+              spec:
+                accessModes: [ "ReadWriteOnce" ]
+                storageClassName: "csi-standard"
+                resources:
+                  requests:
+                    storage: 1Gi
 ```
 
 Here,
 
-- `spec.target.volumeClaimTemplates`:
-  - `metadata.name` is the name of the restored `PVC` or prefix of the `VolumeSnapshot` name.
-  - `spec.dataSource`: `spec.dataSource` specifies the source of the data from where the newly created PVC will be initialized. It requires following fields to be set:
-    - `apiGroup` is the group for resource being referenced. Now, Kubernetes supports only `snapshot.storage.k8s.io`.
-    - `kind` is resource of the kind being referenced. Now, Kubernetes supports only `VolumeSnapshot`.
-    - `name` is the `VolumeSnapshot` resource name. In `RestoreSession` crd, You must set the VolumeSnapshot name directly.
+- `spec.addon.targetVolumes.volumeClaimTemplates[*]`:
+  - `metadata.name` is a template for the name of the restored `PVC`. KubeStash will create `PVC` with the specified name. 
```

Comment:
description for datasource field?

`spec.dataSource.repository`: specifies name of the `Repository` from which the data will be restored. 
`spec.dataSource.snapshot`: specifies the name of the `Snapshot` that will be restored.

Reply:
> applies to all the relevant places